event-buffer/event-buffer-core/src/main/java/uk/gov/justice/services/event/buffer/core/service/ConsecutiveEventBufferService.java

```diff
@@ -91,7 +92,7 @@ private long versionOf(final JsonEnvelope event) {
     }
 
     private boolean smallerThanInitial(final long incomingEventVersion) {
-        return incomingEventVersion < INITIAL_VERSION;
+        return incomingEventVersion < (INITIAL_VERSION + 1);
```

Comment:
this could be "incomingEventVersion == INITIAL_VERSION" and then the method could be in lined